ludwig/distributed/ddp.py

```diff
@@ -128,8 +130,12 @@ def create_checkpoint_handle(
 
         return MultiNodeCheckpoint(self, model, optimizer, scheduler)
 
-    def to_device(self, model: nn.Module, device: Optional[torch.device] = None) -> nn.Module:
-        return model
+    def to_device(self, model: "BaseModel", device: Optional[torch.device] = None) -> nn.Module:
```

Comment:
nit: Do we want to add `model: Union["BaseModel", "DistributedDataParallel"]`

Reply:
Fixed.